src/wolf/wolf.cpp

```diff
@@ -246,6 +246,19 @@ auto setup_sessions_handlers(const immer::box<state::AppState> &app_state,
             wl_promise->set_value(std::move(wl_state));
           }
 
+          /* Adding custom state folder */
+          mounted_paths.push_back({session->app_state_folder, "/home/retro"});
+
+          /* nvidia needs some extra paths */
+          if (session->app->h264_encoder == state::NVIDIA || session->app->hevc_encoder == state::NVIDIA) {
+            mounted_paths.push_back({get_env("NVIDIA_DRIVER_VOLUME_NAME", "nvidia-driver-vol"), "/usr/nvidia"});
+            full_devices.push_back("/dev/nvidia0");
+            full_devices.push_back("/dev/nvidia-modeset");
+            full_devices.push_back("/dev/nvidia-uvm");
+            full_devices.push_back("/dev/nvidia-uvm-tools");
+            full_devices.push_back("/dev/nvidiactl");
+          }
```

Comment:
Just because you are encoding with an nvidia devices, you don't have to render on an nvidia device. Also you might have multiple nvidia devices (and need to map the corresponding `/dev/nvidia*` device).

Can we please not force this but make it optional?

Reply:
Both very valid points, I knew I had to move it behind a flag or something.  
The idea is to try to avoid having users to mess with the config file and provide some sensible default, in this case, since we know that nvidia is working we can safely add those.

I agree with your point, though. I'll think of a better way to (de)activate this mode!

Reply:
I guess this plays into being able to set a custom render-node per app? So we could use the encoding-device as a default and overriding the render-node would disable automatic mapping or invoke different logic to determine the vendor of the node.